lib/datev/export.rb

```diff
@@ -18,7 +18,7 @@ def <<(attributes)
     def to_s
       CSV.generate(CSV_OPTIONS) do |csv|
         write(csv)
-      end
+      end.encode(Encoding::WINDOWS_1252)
```

Comment:
I've just run the specs with Ruby 2.0.0p648 on OS X - no problems with AND without your change. So, until we are not sure that the change is really needed I prefer to remove it.